kernels/volk/volk_32fc_x2_square_dist_32f.h

```diff
@@ -392,7 +388,7 @@ volk_32fc_x2_square_dist_32f_u_avx2(float* target, lv_32fc_t* src0, lv_32fc_t* p
     target += 4;
   }
 
-  for(i = 0; i < leftovers1; ++i) {
+  if (num_bytes >> 3 & 0b11) {
```

Comment:
That's probably the line that breaks the CI. `0b11` may result in `leftovers1 == 3`. Without the loop, Only the first of those 3 samples is processed. And the test complains that the last 2 samples are `0` instead of the correct value.

Reply:
You were right))